services/agent/src/simcore_service_agent/core/settings.py

```diff
@@ -26,8 +26,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
     AGENT_VOLUMES_CLEANUP_TARGET_SWARM_STACK_NAME: str = Field(
         ..., description="Exactly the same as director-v2's `SWARM_STACK_NAME` env var"
     )
-    AGENT_VOLUMES_CLEANUP_S3_SECURE: bool = False
-    AGENT_VOLUMES_CLEANUP_S3_ENDPOINT: str
+    AGENT_VOLUMES_CLEANUP_S3_ENDPOINT: AnyHttpUrl | None
```

Comment:
when this is nullable it's considered to be S3 AWS?
So I guess rlcone knows that an empty S3 endpoint is AWS.

Please addd a validator, where if `AGENT_VOLUMES_CLEANUP_S3_ENDPOINT is None`  `AGENT_VOLUMES_CLEANUP_S3_PROVIDER` has to be set to `AWS`

Reply:
I was starting to do it, but actually that is not correct. for example if I use the Moto library which emulates AWS, the endpoint cannot be None.